libs/orchestrator-ui-components/src/lib/utils/date.ts

```diff
@@ -7,7 +7,7 @@ export const parseDate = (date: string | null | undefined): Date | null => {
     }
 
     // Todo #95: GraphQL Dates will soon be in ISO8601 format instead of a float in a string
```

Comment:
Is the todo still valid?

Reply:
It is partially valid still, since the harcoded "dutch locale" mentioned in bug 95, not tackled yet

Reply:
Yup but that will probably be tackled in Ruben localisation ticket.
Better to remove the Todo: as dates are now indeed IS)8601 formatted

Reply:
done